### ISSUE TITLE
us_vt enum errors

### ISSUE THREAD

**Issue by colincadams** [Recidiviz/recidiviz-data#945]
('us_vt', 'BondType', 'CASH OR BOND'),
 ('us_vt', 'ChargeStatus', 'DEFSTD'),
 ('us_vt', 'ChargeStatus', 'DETAINED'),
 ('us_vt', 'ChargeStatus', 'INVESTIGATION'),
 ('us_vt', 'ChargeStatus', 'P'),
 ('us_vt', 'ChargeStatus', 'SPLIT'),
 ('us_vt', 'ChargeStatus', 'STDPRO'),
 ('us_vt', 'CourtType', 'ANCR'),
 ('us_vt', 'CourtType', 'BNCR'),
 ('us_vt', 'CourtType', 'CNCR'),
 ('us_vt', 'CourtType', 'DEFERRED'),
 ('us_vt', 'CourtType', 'EXCR'),
 ('us_vt', 'CourtType', 'FRCR'),
 ('us_vt', 'CourtType', 'LECR'),
 ('us_vt', 'CourtType', 'OECR'),
 ('us_vt', 'CourtType', 'OOS'),
 ('us_vt', 'CourtType', 'OSCR'),
 ('us_vt', 'CourtType', 'RDCR'),
 ('us_vt', 'CourtType', 'TERM'),
 ('us_vt', 'CourtType', 'WMCR'),
 ('us_vt', 'CourtType', 'WNCR'),
 ('us_vt', 'CourtType', 'WRCR'),
 ('us_vt', 'Gender', 'T'),
 ('us_vt', 'Race', 'A'),
 ('us_vt', 'Race', 'AMERICAN INDIAN OR ALASKAN NATIVE')

**Comment by jessex** [comment]
All of those court type acronyms may be an interesting edge case: when a field we've modeled as an enum may not actually be as enumerable as we think it is, i.e. is a bit more dynamic in some regions. All of those court type codes might be a tightly bound, slowly expanding set, but to me they look kind of like codes which we'd need to deduce the formula for to write them all out. I wonder if we need a more dynamic way of handling some of the enum overrides, e.g. with functions instead of key-value pairs?

**Comment by colincadams** [comment]
Yeah we discussed using functions as part of the solution for #816 which @jamwalla just sent the PR out for!

**Comment by colincadams** [comment]
Reassinging to @ohinds since he seems to handle enum mappings these days :grin:
